support/http/logging_middleware_test.go

```diff
@@ -11,6 +11,21 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+// set "X-Forwarded-For" header to test LoggingMiddlewareWithOptions
+func setXFFMiddleWare(next stdhttp.Handler) stdhttp.Handler {
```

Comment:
nit: method documentation usually starts with the method name and ends with a period `.`

You can see some examples at https://blog.golang.org/godoc.

```suggestion
// setXFFMiddleware sets "X-Forwarded-For" header to test LoggingMiddlewareWithOptions
func setXFFMiddleware(next stdhttp.Handler) stdhttp.Handler {
```

Also, Middleware is a single word so you don't need the capital `W` as in MiddleWare.

Reply:
Addressed here a18f383